core/src/test/resources/HistoryServerExpectations/completed_app_list_json_expectation.json

```diff
@@ -22,6 +23,7 @@
     "duration" : 101795,
     "sparkUser" : "jose",
     "completed" : true,
+    "appSparkVersion" : "",
```

Comment:
I think I know what this is. `FsHistoryProvider` has this code to speed up the log parsing for listings:

```
  protected def mergeApplicationListing(fileStatus: FileStatus): Unit = {
    val newAttempts = try {
      val eventsFilter: ReplayEventsFilter = { eventString =>
        eventString.startsWith(APPL_START_EVENT_PREFIX) ||
          eventString.startsWith(APPL_END_EVENT_PREFIX)
      }
```

It only allows those two events through currently, you probably need to the log start event to that list too.